a10_octavia/controller/worker/tasks/a10_network_tasks.py

```diff
@@ -704,6 +704,20 @@ def _remove_device_vrid_fip(self, partition_name, vrid_value):
                           str(vrid_value))
             raise e
 
+    def _create_vrid(self, vrid):
+        has_vrid = True
+        try:
+            self.axapi_client.vrrpa.get(vrid)
+        except acos_errors.NotFound:
+            has_vrid = False
+
+        try:
+            if not has_vrid:
```

Comment:
Why not add this block of code in above exception.

Reply:
A better method would be to add an `exists` function in acos-client (example: https://github.com/a10networks/acos-client/blob/6df217774ee94912a9760741c9271fa4d8abc7af/acos_client/v30/file/ssl_key.py#L30). Then we can do the following:

```
try:
    if not self.axapi_client.vrrpa.exists(vrid):
        self.axapi_client.system.action.configureVRID(vrid)
except Exception as e:
   LOG.exception("Failed to create vrid d: %s", vrid, str(e))
   raise e
```